.wordlist.txt

```diff
@@ -4,6 +4,7 @@ AAR
 AArch
 ABI
 ABIs
+ACLs
 ADB
 addon
 addons
```

Comment:
Need to add "customisations" (and probably add "customisation" at the same time, even though we don't need it right now, it's the correct spelling).

Reply:
So the spellcheck doesn't recognize the correct spelling?

Reply:
Exactly. The spellcheck doesn't know the word "customisation" (or "customization", for that matter).